Client/Frontend/Browser/HomePanel/libarchive.h

```diff
@@ -0,0 +1,152 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+//From https://github.com/libarchive/libarchive/blob/master/libarchive/archive.h
```

Comment:
Move this file to `ThirdParty` folder is possible.
https://github.com/brave/brave-ios/tree/development/ThirdParty